README.md

```diff
@@ -41,6 +41,18 @@ Next, we will need to install `wasm-bindgen-cli`. In the future Trunk will handl
 cargo install wasm-bindgen-cli
 ```
 
+In addition, we will need to install `wasm-opt` which is part of `binaryen`. On MacOS we can install
```

Comment:
Can we bundle the binary with trunk or download on command invocation (and cache it)?

Reply:
As I wrote in the description I though of adding that as a separate PR because it would involve `wasm-bindgen-cli` as well so we don't need any binaries present.

And there is #105 which would require to re-write download part again as the HTTP client would change as well.